README.md

```
@@ -106,15 +106,18 @@ class MyModelCrudl(CrudlController[MyModel]):
+            mode="hard",  # or "soft" if you implement soft_delete()
+        )
     )
 ```
 
 **NOTE:** In order to avoid accidentally exposing sensitive fields, you need to explicitly define the model fields that shall be exposed via the CRUDL endpoints. Some other libraries support exposing all fields (with optional exclude) which can lead to unintentional exposure of sensitive data.
 
-**NOTE:** If any of `create_schema`, `update_schema`, `get_one_schema`, or `list_schema` are not defined or are set as `None`, then that specific endpoint will not be exposed. If `delete_allowed` is not defined or set as `False`, then the delete endpoint will not be exposed.
+**NOTE:** If any of `create_schema`, `update_schema`, `get_one_schema`, or `list_schema` are not defined or are set as `None`, then that specific endpoint will not be exposed. If `delete_options` is not defined or `allowed` is set as `False`, then the delete endpoint will not be exposed.
 
-**NOTE:** For delete operation, it currently performs a hard delete by default. You might customize the delete operation to perform a soft delete by [overriding the delete method in the model]().
+**NOTE:** For delete operation, it currently performs a hard delete by default. You might customize the delete operation to perform a soft delete by [overriding the delete method in the model]() or **alternatively** by setting `mode="soft"` in the `DeleteOptions` and implementing the `soft_delete()` method in your model CRUDL config.
```

markdownlint-cli2 (0.18.1)

No empty links

(MD042, no-empty-links)